javascript/ql/integration-tests/all-platforms/no-types/test.py

```diff
@@ -0,0 +1,3 @@
+from create_database_utils import *
+
+run_codeql_database_create([], lang="javascript", extra_env={"CODEQL_EXTRACTOR_JAVASCRIPT_OPTION_SKIP_TYPES": "true"})
```

Comment:
Perhaps pass the `-Oskip_types=true` flag instead of setting the environment variable. That should in addition test that the extractor option works.

Reply:
Yep, that works 👍 

But it only works after merging the internal PR.  
Which just confirms that the `codeql-extractor.yml` changes are needed.